kaldb/src/test/java/com/slack/kaldb/server/KaldbTest.java

```diff
@@ -352,7 +426,8 @@ public void testTwoIndexersAndOneQueryService() throws Exception {
             KALDB_TEST_CLIENT_2,
             ZK_PATH_PREFIX,
             2,
-            startTime2);
+            startTime2,
+            indexer2MeterRegistry);
     indexer2.serviceManager.awaitHealthy(DEFAULT_START_STOP_DURATION);
 
     KaldbSearch.SearchResult indexerSearchResponse =
```

Comment:
I think it would be good to have 2 more tests when the same chunk is on multiple nodes:
* The same chunk is on indexer and cache node.
* The same chunk is on multiple cache nodes.

Reply:
I think these tests exist on KaldbDistributedQueryServiceTest.java